src/network/networkctl.c

```diff
@@ -691,14 +705,18 @@ static int list_links(int argc, char *argv[], void *userdata) {
         (void) table_set_ellipsize_percent(table, cell, 100);
 
         for (int i = 0; i < c; i++) {
-                _cleanup_free_ char *setup_state = NULL, *operational_state = NULL;
+                _cleanup_free_ char *setup_state = NULL, *operational_state = NULL, *online_state = NULL;
                 const char *on_color_operational, *off_color_operational,
+                           *on_color_online, *off_color_online,
```

Comment:
`off_color_online` is not used. See #19288.

Reply:
I'll rebase and fix once #19288 is merged.